src/main/groovy/com/spidasoftware/schema/conversion/changeset/v9/EnvironmentClientDataChangeset.groovy

```diff
@@ -6,9 +6,6 @@ package com.spidasoftware.schema.conversion.changeset.v9
 import com.spidasoftware.schema.conversion.changeset.ConversionException
 import com.spidasoftware.schema.conversion.changeset.client.AbstractClientDataChangeSet
 
-import groovy.transform.CompileStatic
-
-@CompileStatic
```

Comment:
what was the actual problem and how does this fix it?

Reply:
We think the CompileStatic with JDK 7's invokedynamic was cutting out certain methods when compiling. We were getting a NoSuchMethodError for applyPrettyPrintToEnvironment and getDefaultEnvironmentMaps with a list's .each and .sort respectively.